test/translation/lua/enum.lua

```diff
@@ -1,4 +1,5 @@
 TestEnum={}
 TestEnum.val1=0
 TestEnum.val2=2
-TestEnum.val3=3
\ No newline at end of file
+TestEnum.val3=3
+TestEnum.val4="bye"
```

Comment:
Revert this file back to just a generic enum as it was before, this case is covered by enumHeterogeneous.lua